tests/dummy/app/templates/list-controls.hbs

```diff
@@ -28,7 +28,7 @@
         {{#paper-subheader class="md-no-sticky"}}Clickable Items with Secondary Controls{{/paper-subheader}}
 
         {{#paper-item action="transitionToWifiMenu"}}
-          {{paper-icon "network-wifi"}}
```

Comment:
See https://github.com/google/material-design-icons/blob/master/iconfont/codepoints , this is valid names. I could not get it working with dashes.